examples/service-sameness/README.md

```diff
@@ -0,0 +1,276 @@
+dc2_server_url = "http://ecs-dc2-consul-server-1078624692.us-west-2.elb.amazonaws.com:8500"
+```
+
+### Explore
+
+Get the bootstrap token for both the clusters from the Terraform output:
+
+```console
+$ terraform output -json | jq -r .dc1_server_bootstrap_token.value
+e2cb39e2-b9fd-18af-025f-86f6da6889a7
+
+$ terraform output -json | jq -r .dc2_server_bootstrap_token.value
+e2cb39e2-b9fd-18af-025f-6f63da6882a7
+```
+
+If you click on the URL of the `dc1_server_url`, you should be able
+to view the Consul UI and log in using the `dc1_server_bootstrap_token` above:
+
+![Consul dc1 UI](../../_docs/sameness-dc1-ui.png?raw=true)
+
+If you click on the `part-1` partition under the partitions dropdown, you should see the following UI:
+
+![Consul dc1 part1 UI](../../_docs/sameness-dc1-part1-ui.png?raw=true)
+
+If you click on the URL of the `dc2_server_url`, you should be able
+to view the Consul UI and log in using the `dc2_server_bootstrap_token` above:
+
+![Consul dc2 UI](../../_docs/sameness-dc2-ui.png?raw=true)
+
+If you click on the Peers subsection in the left pane in `dc2_server_url` you should see that
+a successful peering connection has been established with both the `default` and `part-1` partitions of the `dc1` cluster.
+
+![Consul dc2 peering](../../_docs/sameness-peering-dc2.png?raw=true)
+
+Similarly you should be able to see that `dc2-cluster` is a peer for `dc1`'s default and `part-1` partition from `dc1_server_url`. You should also see that the example server app in `dc2` is exported to both the partitions of `dc1` via the peering connection. This can be seen by clicking on the peer name in `dc1_server_url` under the `Peers` subsection.
+
+![Consul dc1 exported service](../../_docs/sameness-exported-service-dc1.png?raw=true)
+
+If you browse to the URL of the `client.lb_address` of the apps present in the terraform output, the example application UI should be displayed:
+
+![Example App UI](../../_docs/sameness-client-ui.png?raw=true)
+
+You should notice that all the client apps at this point will dial the server apps present in the same partition via their envoy sidecar. 
+
+## Sameness Group illustration
+
+We will now manually scaledown some of these server apps and visualize the changes to illustrate the failover usecases with Consul's service sameness groups. You can either try this manually by following the upcoming sections or run the `sameness-test` script to automate it.
+
+With all the server apps functioning as expected, the calls from client apps should reach the server apps present in their local partition.
+
+![Sameness Demo 1](../../_docs/sameness-demo-1.png?raw=true)
+
+### Step 1
+
+Manually scale down the server app present in DC1's `default` partition with the following command and wait for a few seconds for the change to propagate to Consul
+
+```console
+aws ecs update-service --region ${AWS_REGION} --cluster ${DC1_DEFAULT_PARTITION_ECS_CLUSTER_ARN} --service ecs-dc1-example-server-app --desired-count 0
```

Comment:
We should provide some commands that set up the environment variables referenced in the commands:
- AWS_REGION
- DC1_DEFAULT_PARTITION_ECS_CLUSTER_ARN
- DC1_PART1_PARTITION_ECS_CLUSTER_ARN
- DC2_DEFAULT_PARTITION_ECS_CLUSTER_ARN
- DC1_PART1_PARTITION_ECS_CLUSTER_ARN

```
export AWS_REGION=us-west-2
export DC1_DEFAULT_PARTITION_ECS_CLUSTER_ARN=$(terraform output -json | jq -r .dc1_default_partition_apps.value.ecs_cluster_arn)
...
```

Reply:
Added commands to set these up.